blocks/right-rail-block/index.story.jsx

```diff
@@ -30,18 +36,57 @@ const layoutItemStyles = {
 		textAlign: "center",
 	},
 };
-const getStyles = (name) => {
-	const defaults = layoutItemStyles.default;
 
-	return { ...defaults, ...layoutItemStyles[name] };
+const layoutItemStylesCommerce = {
+	navigation: {
+		backgroundColor: "black",
+		color: "white",
+		height: "100px",
+		display: "flex",
+		flexDirection: "column",
+		justifyContent: "center",
+		alignItems: "center",
+	},
+	footer: {
+		height: "300px",
+	},
+	fullWidth2: {
+		height: "250px",
+	},
+	main: {
+		height: "800px",
+	},
+	rightRail: {
+		height: "600px",
+	},
+	default: {
+		width: "100%",
+		backgroundColor: "rgb(230 230 230)",
+		textAlign: "center",
+	},
 };
 
-const layoutItem = (name) => <div style={getStyles(name)}>{name}</div>;
+const layoutItemBasic = (name) => <div style={getStyles(name, layoutItemStylesBasic)}>{name}</div>;
 
-const layoutAreas = ["navigation", "fullWidth1", "main", "rightRail", "fullWidth2", "footer"];
+const layoutItemCommerce = (name) => (
+	<div style={getStyles(name, layoutItemStylesCommerce)}>
+		{name}
+		{name === "navigation" ? <h3>Use the commerce theme when viewing this story</h3> : null}
```

Comment:
Take a look at my WIP PR on how we can address this - https://github.com/WPMedia/arc-themes-blocks/pull/1425/files#r921992253

Reply:
@matthewroach Brilliant!  I added the logic to this branch.